fsspec/asyn.py

```diff
@@ -261,13 +266,13 @@ async def _put(self, lpath, rpath, recursive=False, **kwargs):
                 self._put_file(lpath, rpath, **kwargs)
                 for lpath, rpath in zip(lpaths, rpaths)
             ],
-            disable=self.disable_throttling,
+            batch_size=batch_size,
         )
 
     async def _get_file(self, rpath, lpath, **kwargs):
         raise NotImplementedError
 
-    async def _get(self, rpath, lpath, recursive=False, **kwargs):
+    async def _get(self, rpath, lpath, recursive=False, batch_size=None, **kwargs):
```

Comment:
Sorry, I meant in the actual configuration, `fsspec.config`, a key like "async_gather_batch_size". If defined, it would be used instead of  _DEFAULT_BATCH_SIZE.

The version here would need to have the signature changed in any subclass that overrides this method.